app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,15 @@
   <head>
     <title>ScootsShop</title>
     <%= csrf_meta_tags %>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+		<!-- Website CSS style -->
+		<link href="css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Not quite sure about importing bootstrap again here, but if it works for now that's cool. We'll try playing around with it at a later date.